.claude/skills/preflight/scripts/resolve-prior-threads.sh

```
@@ -0,0 +1,137 @@
+  # Post reply noting the resolving commit
+  gh api "repos/$owner/$repo/pulls/$pr_number/comments/$database_id/replies" \
+    -f body="Resolved — addressed in \`$sha\`." >/dev/null 2>&1 || true
+
+  # Collapse the thread via GraphQL
+  gh api graphql -f query='
+    mutation($id: ID!) {
+      resolveReviewThread(input: {threadId: $id}) {
+        thread { isResolved }
+      }
+    }
+  ' -f id="$thread_id" >/dev/null 2>&1 || true
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)